tests/fixtures/ssl/Makefile

```diff
@@ -0,0 +1,12 @@
+CA_PASSWORD?=strong_password
```

Comment:
Is it really a strong password? :)

Reply:
It was just and advice to use a strong password instead of that. I think it's more readable than a real strong password